WebView/app/src/main/assets/index.html

```diff
@@ -0,0 +1,33 @@
+<!--
+  ~ Copyright (C) 2020 The Android Open Source Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="app.css" rel="stylesheet">
+        <title></title>
+    </head>
+    <body onload="getData()">
+        <h1 id="location">Today's Weather</h1>
+        <h2 class="day">Monday</h2><img id= "icon" alt="weather" class="center" src="https://gcoleman799.github.io/res/drawable/sunny.png">
+        <h2 id="description">Sunny and 75</h2>
+        <form>
```

Comment:
Do we need a <form> or can we include the button without this?

Reply:
Yes, I removed form